docs/summary-data.md

```diff
@@ -74,7 +74,7 @@ All of the data is stored within each user account to enable quick sorting and f
 
 ## Threshold Values
 
-The summary calculation uses the glycemic targets established by [ADA](https://diabetes.org/) [[standards of care](https://diabetesjournals.org/care/issue/46/Supplement_1)] and [AACE](https://pro.aace.com/) [[paper](https://doi.org/10.1016/j.eprac.2022.08.002), [table](https://www.endocrinepractice.org/article/S1530-891X(22)00576-6/fulltext#tbl6)] to characterize each `cbg` or `smbg` glucose value as one of very low, low, in range, high, or very high. The same target ranges are currently used for all users, and not personalized based on the user's diagnosis type or either the user's or the clinic's preferences. The glycemic target ranges are:
+The summary calculation uses the glycemic targets established by [ADA](https://diabetes.org/) [[standards of care](https://diabetesjournals.org/care/issue/46/Supplement_1)] and [AACE](https://pro.aace.com/) [[paper](https://doi.org/10.1016/j.eprac.2022.08.002), <!-- markdown-link-check-disable-line -->[table](https://www.endocrinepractice.org/article/S1530-891X(22)00576-6/fulltext#tbl6)] to characterize each `cbg` or `smbg` glucose value as one of very low, low, in range, high, or very high. The same target ranges are currently used for all users, and not personalized based on the user's diagnosis type or either the user's or the clinic's preferences. The glycemic target ranges are:
```

Comment:
This is fine for now, though I think it disables the checks for the entire line in the Markdown (i.e. all the links in this paragraph).

Ultimately we can replace this with a JSON config file like https://github.com/tcort/markdown-link-check that just excludes that one troublesome domain:

```
{
  "ignorePatterns": [
    {
      "pattern": "^https://www.endocrinepractice.org"
    }
  ]
}
```

Reply:
updated as per comment @tjotala. As mentioned at least now its included and the config can be updated for any other exceptions